src/ebay-education-notice/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import EbayIcon from '../../ebay-icon/icon'
+import {
+    EbayEducationNotice,
+    EbayNoticeContent,
+    EbayEducationNoticeTitle,
+    EbayEducationNoticeFooter
+} from '../index'
+import userEvent from '@testing-library/user-event'
+import { act } from 'react-dom/test-utils'
+
+describe('<EbayEducationNotice>', () => {
+    it('should render notice snapshot', () => {
+        const { asFragment } = render(
+            <EbayEducationNotice
+                a11yIconText="a11y notice"
+                educationIcon={<EbayIcon name="theEbayVault24" />}
+            >
+                <EbayEducationNoticeTitle>
+                    Recommended title format
+                </EbayEducationNoticeTitle>
+                <EbayNoticeContent>
+                    <p>
+                        Follow the order below to optimize market valuation from
+                        Price Guide. Player + Set or Season + Manufacturer +
+                        Card number + Variant + Grader + Grade
+                    </p>
+                </EbayNoticeContent>
+                <EbayEducationNoticeFooter>
+                    Education footer
+                </EbayEducationNoticeFooter>
+            </EbayEducationNotice>
+        )
+
+        expect(asFragment()).toMatchSnapshot()
```

Comment:
we usually do snapshot testing inside a render.spec.tsx file, you can take the accordion as [an example](https://github.com/eBay/ebayui-core-react/blob/main/src/ebay-accordion/__tests__/render.spec.tsx). We usually uses the stories and do snaphot testing based on those.

For behavior testing (like callback calls, and this case the dismiss behavior) we use the index.spec.tsx file